mapbox/libjava-services/src/main/java/com/mapbox/services/api/staticimage/v1/models/StaticMarkerAnnotation.java

```diff
@@ -144,12 +149,17 @@ public Builder setName(String name) {
      * @since 2.1.0
      */
     public String getLabel() {
-      return label;
+      if (label.isEmpty()) {
+        return EMPTY;
```

Comment:
@cammace `null` is not valid because `getLabel` is called when parsing (`String.format`) and `label` is an optional param (note that `-`was removed from the default string because if the `label` was empty url was incorrect. A [Null Object pattern](https://sourcemaking.com/design_patterns/null_object) could be implemented though.

BTW `null` is considered _The worst mistake of computer science_: [Null References: The Billion Dollar Mistake](https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare) 😉